### PR TITLE
Update to the newest version of Shimming Toolbox [9c454d7]

### LINKED ISSUE #25: Automatically load  NIfTI files in FSLeyes after conversion
### Description of Problem / Inefficiency:
We should automatically load  the NIfTI files created by `st_dicom_to_nifti` in FSLeyes after the conversion, so that the user doesn't have to manually load them in the overlay list.

### Implementation Suggestion:
Go through the `sub-XYZ/fmap` and `sub-XYZ/rfmap` folders and load all the .nii.gz files to the overlay.

### LINKED ISSUE #26: ST logs are not piped to the FSLeyes plugin's terminal
### Bug Description
We log and print a bunch of info during the normal execution of shimming-toolbox, however they are not displaying when shimming-toolbox is called from the FSLeyes GUI.

### PR DESCRIPTION
**Pull request by po09i**
## Description
This updates the plugin to add the new feature of shimming toolbox version: [8be1560](https://github.com/shimming-toolbox/shimming-toolbox/commit/8be1560f7f305c4fe564d35fd8a0b7f6eff29176)

In more details, the PR:
- adds `st_b0shim static` in the shim tab
- adds `st_b0shim realtime` in the shim tab
- refactors to a scrollable tab panel (Very minor)
- implements a way to have different button function if there are multiple text boxes in the same `TextWithButton`
- Automatically load converted NIfTIs into the overlay
- Log the output of the CLIs into the fsleyes terminal

Fixes #25
Fixes #26

### CODE REVIEW

**Comment by gaspardcereza** [comment]
Is this PR still a draft or should it be reviewed @po09i ?

**Comment by po09i** [comment]
Yes It was still a draft because the fix for #21 is in PR #22

**Comment by gaspardcereza** [comment]
@po09i I have [this PR](https://github.com/shimming-toolbox/shimming-toolbox/pull/344) on shimming-toolbox that needs to be merged for the GUI to work so we will need to update the version of shimming-toolbox once it's done.

File: Makefile [hunk 1]
```diff
@@ -21,8 +21,8 @@ install: ## Run 'make install' to install the plugin [Use flag CLEAN=true for cl
 		echo "$(ST_DIR) and conda install found, skipping install of conda"; \
         fi
 
-	bash installer/create_venv.sh
-	bash installer/install_plugin.sh
+	#bash installer/create_venv.sh
+	#bash installer/install_plugin.sh
```
Comment (gaspardcereza) [inline Makefile]: remove

Comment (po09i) [reply Makefile]: Addressed here: [4486a67](https://github.com/shimming-toolbox/fsleyes-plugin-shimming-toolbox/pull/24/commits/4486a671c9f936009bf14837256d5f387ecf9b6b)

**Review by gaspardcereza — APPROVED** [review]
I did not test it but we went through it together and the code looks good.

**Comment by gaspardcereza** [comment]
I did not test it but it looks good.

**Comment by gaspardcereza** [comment]
@po09i merged this for today experiment

**Pull request merged**